src/thriveopedia/fossilisation/MuseumCard.cs

```diff
@@ -88,19 +107,47 @@ private void UpdatePreviewImage()
 
     private void OnPressed()
     {
+        // Make sure clicking doesn't pass through from the delete button
+        if (deleteButton?.Pressed ?? false)
+        {
+            Pressed = wasPressed;
+            return;
+        }
+
+        wasPressed = Pressed;
+
         GUICommon.Instance.PlayButtonPressSound();
         EmitSignal(nameof(OnSpeciesSelected), this);
     }
 
     private void OnMouseEnter()
     {
+        if (deleteButton != null)
+            deleteButton.Visible = true;
+
         GUICommon.Instance.Tween.InterpolateProperty(speciesPreview, "modulate", null, Colors.Gray, 0.5f);
         GUICommon.Instance.Tween.Start();
     }
 
     private void OnMouseExit()
     {
+        if (deleteButton != null)
+            deleteButton.Visible = false;
+
         GUICommon.Instance.Tween.InterpolateProperty(speciesPreview, "modulate", null, Colors.White, 0.5f);
         GUICommon.Instance.Tween.Start();
     }
+
+    private void OnDeletePressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        deleteConfirmationDialog.DialogText = TranslationServer.Translate("DELETE_FOSSIL_CONFIRMATION");
```

Comment:
Why not add the text directly through the inspector rather than code?

Reply:
Copied how `ThriveopediaMuseumPage.OnOpenInFreebuildPressed()` did it, I can change that

Reply:
Fixed